### ISSUE TITLE
Windows 11 23H2 / 22H2

### ISSUE THREAD

**Issue by dougg0k** [dougg0k/gaming_os_tweaker#9]
I will probably move to Win11 once the update release and see how the tweaks will behave in there. There were improvements done to Win11 these last updates that may have improved latency and performance. Also had mouse pooling improvements.

23H2 will probably be released before the end of 2023 - September or after.

Update README once it's done and tested and add a update note.

**Comment by dougg0k** [comment]
I already did some testing, fixed and improved a bunch of scripts. Though there are still some issues not resolved in Win11 22H2.

Services not disabling
WinDefend
WdNisSvc

Tasks from Task Scheduler related to UpdateOrchestrator are not removable - Access is Denied

**Comment by dougg0k** [comment]
Fixed infinite boot local session manager msg - https://github.com/dougg0k/gaming_os_tweaker/commit/a11bee1006b1e6925eae59bd1784f00eb4e061fa

Still the other issues are not resolved.

As far as performance, seems very similar to Windows 10 after all tweaks. DPC avg also close, but a bit more.

Another problem that I had seen before and was not able to fix. For whatever reason in Overwatch 2, you stay with the fps locked to your refresh rate in fullscreen or borderless fullscreen, only in windowed mode it unlocks and reaches the max you have set.

**Comment by dougg0k** [comment]
I tried NanaRun which is called MiniSudo aka new version of NSudo. Also tried SetACL. None have worked.

I found one pattern to Overwatch2 fps lock, if I disable Optimized for windowed games, I can run in Borderless Fullscreen and Window mode with the fps that I set, but Fullscreen mode are still being locked. However, if I enable, I can only have the fps fully unlocked to where I set in Window mode, where both other options are locked to the refresh rate regardless. 

Not sure why they would do this, but it's clearly broken and not only with Overwatch.

https://www.reddit.com/r/blackdesertonline/comments/xq9sq6/for_those_interested_win11_22h2_is_now_available/
https://www.reddit.com/r/blackdesertonline/comments/xk7ixg/psa_windows_11_users_may_now_uncap_fps_in/

**Comment by dougg0k** [comment]
Only way I found in how to disable Windows Defender services are to enter safemode and execute the scripts as admin. Not even with the very complex script from https://privacy.sexy/ I was not able to disable it in normal mode.

**Comment by dougg0k** [comment]
Not sure if this is the effect of not having Windows Defender, but the first time I tried, I was using MinSudo incorrectly, now I have used it correctly and I can alter/remove everything. All issues mentioned in this issue are now dealt with. https://github.com/dougg0k/gaming_os_tweaker/commit/31bc014946051cb1ba2365f5e6b7f5b83616b703

I even removed the need of a tool and started using MinSudo instead much more clean, I could abstract, but wont now.

All issues besides the one that is related to having to activate gsync to unlock fps, but that might be just from game devs. https://github.com/microsoft/DirectXTK/issues/395
